frontend/views/containers/payments/PaymentRowReceived.vue

```diff
@@ -111,6 +115,9 @@ export default ({
         console.error(e)
         alert(e.message)
       }
+    },
+    async updatePayment () {
+      this.payment.relativeTo = await this.getPeriodStampGivenDate(this.payment.date)
```

Comment:
I think since the `relativeTo` is reactive to the `payment` prop, we shouldn't set it only in the `created` lifecycle. Since the function `getPeriodStampGivenData` is async function, we could setup watch for `payment` prop.

```
data () {
    relativeTo: undefined
},
...
watch: {
    payment (to, from) {
        this.updatePaymentRelativeTo(to.date)
    }
},
methods: {
    async updatePaymentRelativeTo (paymentDate) {
        this.relativeTo = await this.getPeriodStampGivenData(paymentDate)
    }
}
```